### PR TITLE
Add support for cloud events output

### PR DESCRIPTION
**Pull request by nsmith5**
Adds basic support for cloud event output. My understanding is a little superficial on this subject so looking for guidance on if this implementation is reasonable or has any limitations before merging

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,3 +170,13 @@ outputs:
...
 ```
+
+**cloudevents**
+
+The `cloudevents` driver emits [cloud events](https://cloudevents.io).
+
+```diff
+outputs:
++  cloudevents:
++    url: http://example.com:1234
```
Comment (n3wscott) [inline README.md:181]: I might rename url to http, or have 
```
http:
  url:
```

Then you could support sending to a event broker (like pubsub or amqp) without needing to rework your config.

File: outputs/cloudevents/cloudevents.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+	ce "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/client"
+	"github.com/nsmith5/rekor-sidekick/outputs"
+)
+
+const (
+	driverName  = `cloudevents`
+	eventSource = `github.com/nsmith5/rekor-sidekick`
+	eventType   = `rekor-sidekick/event`
```
Comment (n3wscott) [inline outputs/cloudevents/cloudevents.go:15]: I would rename the event type to be something more descriptive and also with a version, like:

```
eventType   = `rekor.policy.violation.v1`
```

The event is already namespaced to rekor-sidekick in the source, in cloudevents you view the "namespace" of the event in the source+event type tuple.

File: outputs/cloudevents/cloudevents.go [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	"errors"
+
+	ce "github.com/cloudevents/sdk-go/v2"
+	"github.com/cloudevents/sdk-go/v2/client"
+	"github.com/nsmith5/rekor-sidekick/outputs"
+)
+
+const (
+	driverName  = `cloudevents`
+	eventSource = `github.com/nsmith5/rekor-sidekick`
```
Comment (n3wscott) [inline outputs/cloudevents/cloudevents.go:14]: I know I do this in the samples as is coded here, but in real production type systems, eventSource should also be appended with some identifying data to be able to find it. That could be as simple as the server url, or sometimes I make a guid and append it to the source path as you have.

Just something to think about, if you had two of these running and the client was getting events, how would it know which rekor-sidekick was invoking it?

**Review by n3wscott — COMMENTED** [review]
Overall LGTM, some nits on formatting of the cloudevents fields.

File: outputs/cloudevents/cloudevents.go (hunk 3 above)
Comment (nsmith5) [reply outputs/cloudevents/cloudevents.go:14]: Ahhh I see. There is no unique information to making routing policy on...

**Pull request merged**